android/src/app/organicmaps/help/HelpFragment.java

```diff
@@ -105,22 +105,22 @@ private void openLink(@NonNull String link)
 
   private void onPrivacyPolicyClick()
   {
-    openLink(getResources().getString(R.string.privacy_policy_url));
+    openLink(getResources().getString(R.string.translated_om_site_url) + "policy/");
   }
 
   private void onTermOfUseClick()
   {
-    openLink(getResources().getString(R.string.terms_of_use_url));
+    openLink(getResources().getString(R.string.translated_om_site_url) + "terms/");
   }
 
   @Override
   public void onClick(View v)
   {
     final int id = v.getId();
     if (id == R.id.web)
-      openLink(Constants.Url.WEB_SITE);
```

Comment:
Remove unused Constants.Url entries please.

Reply:
I removed them. Can you please clarify what do you mean?

Reply:
Sorry I missed that hunk among others changes.